src/main/java/core/basesyntax/Bird.java

```diff
@@ -0,0 +1,4 @@
+package core.basesyntax;
+
+public abstract class Bird extends Animal {
```

Comment:
is there a point in completely empty class?

Reply:
added